components/textFields/AdvancedOpenTextField.js

```diff
@@ -30,66 +32,52 @@ const AdvancedOpenTextField = (props) => {
     // Populate if value already exists in redux
     if(!value) {
         if(existingData != null) {
-            if(existingData[currId] != null && !value) {
+            if(existingData[currId] != null && existingData[currId] != '' && !value) {
                 setValue(existingData[currId]);
-                setButtonAppearance('filled');
-                setAdvancedButtonAppearance("filled")
             }
         }
     }
 
-    const submitField = () => {
-      //submits data to reducer and is shown as complete
-        if(!value) {
-            return;
-        }
-        submitFunction({id, question: currId, selection: value})
-        switch(currId) {
-          case "Bw7d2KTr": // VIN question id
-            if (!vinValidator.validate(value)) {
-              setInvalidVin(true);
-            } else {
-              setInvalidVin(false);
-            }
-            break;
-          default:
-            break;
-        }
-        ErrorMsg();
-        setButtonAppearance('filled');
-    }
-
     const onTextChange = (text) => {
         //Updated the reducer when user types
+        var localText = text
```

Comment:
Why do you need to set `localText`? You should be able to just use `text`. All please update all `var` to `let`

Reply:
It's because of the shenanigans we're doing with state changes. I'm able to change the variable name, if that's of preference, but it is relevant to make a local variable here.

Changing var now!